app/forms/curation_concerns/image_form.rb

```diff
@@ -1,7 +1,6 @@
-# Generated via
-#  `rails generate curation_concerns:work Image`
 module CurationConcerns
   class ImageForm < CurationConcerns::Forms::WorkForm
```

Comment:
Just as a warning, this will likely break if #46 is merged prior to this pull request.  I would please suggest that this be merged, and that I work to resolve renaming Image/Raster/Vector to ImageWork/RasterWork/VectorWork within #46.

If an alternative approach is preferred, please let me know.